akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -17,9 +17,14 @@ import akka.http.impl.engine.server.HttpAttributes
 import akka.http.impl.engine.ws.ByteStringSinkProbe
 import akka.http.impl.util.AkkaSpecWithMaterializer
 import akka.http.impl.util.LogByteStringTools
+import akka.http.javadsl.Http
```

Comment:
Is this used?

Reply:
No, probably Intellij, I should probably turn off that autoimport feature it messes with my PRs all the time...